concepts/05 UI Components/List/00 Getting Started with List/05 Group Items.md

```diff
@@ -0,0 +1,116 @@
+The **List** can display items as a two-level hierarchy: parent and child items. To group items, set the [grouped](/Documentation/ApiReference/UI_Components/dxList/Configuration/#grouped) property to **true** and ensure that data is structured as a hierarchical array of objects. Each object must contain a **key** property (parent) and an **items** property (child items). Refer to the following help topic for more information: [Grouping in the Data Source](/Documentation/Guide/UI_Components/List/Grouping/In_the_Data_Source/).
+
+An alternative to the **key** + **items** structure is a flat array grouped using the [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) component. Wrap your array in a [store](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/store/) as shown below and use the DataSource's [group](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#group) property to specify a data field to group data by.
```

Comment:
`grouped using` - it's clear enough, but does break our 'ing' guidelines and is easy to rewrite. 

You also end the sentence with a preposition. Avoid that if possible. 

And smaller sentences also help. 

 ```suggestion
You can also display a hierarchy in a list bound to a flat array. In this case, use the [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) component. Wrap your array in a [store](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/store/) as shown below. Assign the grouping field to the DataSource's [group](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#group) property.
```